Integrations/integration-GoogleSearch.yml

```diff
@@ -0,0 +1,52 @@
+description: Search Google
+configuration:
+- display: Number of pages to retrieve
+  name: pages
+  defaultvalue: "3"
+  type: 0
+  required: true
+script:
+  script: |
+    from google import google
+    # The command demisto.command() holds the command sent from the user.
+    if demisto.command() == "googlesearch":
+        results = google.search(demisto.args()["q"], int(demisto.params()["pages"]))
+        i = 1;
+        ec = {};
+        if (results):
+            md = "### Google results for "+demisto.args()["q"]+"\n"
+            for res in results:
+                md += "#### "+str(i) + " - " + res.name+"\n"
+                md += "["+res.link+"]("+res.link+")\n"
+                md += res.description + "\n\n\n"
+                i += 1
+        else:
+            md = "### No results\n"
+
+        demisto.results( {'ContentsFormat': formats['json'], 'Type': entryTypes['note'], 'Contents': {}, "HumanReadable": md, "EntryContext": ec} );
```

Comment:
You are not using EntryContext - so remove it
Contents - should contain the raw data - in json format, so we can later use extend-context and push data into context